src/ImageSharp/Formats/Jpg/JpegDecoderCore.cs

```diff
@@ -559,42 +547,42 @@ private void ProcessDefineHuffmanTablesMarker(int remaining)
             }
         }
 
-        private void ProcessDefineHuffmanTablesMarkerLoop(ref Huffman huffman, ref int remaining)
+        private void ProcessDefineHuffmanTablesMarkerLoop(ref HuffmanTree huffmanTree, ref int remaining)
```

Comment:
While we're on cleanup, I don't think these refs are needed here. It will be passed by reference not value anyway, the only time you'd want to use ref is if you want to re-assign the object and thus change the reference.